include/GoogleSync/GoogleSync.php

```diff
@@ -495,107 +657,67 @@ public function setGService()
      * @param Meeting|\Google_Service_Calendar_Event $event_1 Either the Meeting Bean or Google_Service_Calendar_Event Object
      * @param \Google_Service_Calendar_Event $event_2 (optional) Google Event
      *
-     * @return string push(_delete), pull(_delete), skip
+     * @return string|bool push(_delete), pull(_delete), skip
```

Comment:
Use typehinting at function parameters, it makes sortEventObjects() unnecessary and more simple code.

Reply:
I'm going to be working on the pushPullSkip method. Give me day or two for testing.

Reply:
The next commit will have a refactored pushPullSkip method.